doc/classes/Geometry3D.xml

```diff
@@ -81,6 +81,24 @@
 			<param index="3" name="c" type="Vector3" />
 			<description>
 				Returns a [Vector3] containing weights based on how close a 3D position ([param point]) is to a triangle's different vertices ([param a], [param b] and [param c]). This is useful for interpolating between the data of different vertices in a triangle. One example use case is using this to smoothly rotate over a mesh instead of relying solely on face normals.
+				[param a], [param b] and [param c] must form a valid triangle and [param point] must be coplanar with the [param a][param b][param c] triangle. Returns an empty [Vector3] if [param a][param b][param c] does not form a valid triangle. Returns invalid weights if [param point] is not coplanar with or exists outside of the bounds of the [param a][param b][param c] triangle.
```

Comment:
For a coplanar point outside of the triangle the returned weights are correct, they'll just won't be in 0-1 range.
Any point coplanar with a triangle is representable as a linear combination of the triangle's vertices (any two edge vectors of a triangle form a basis of the triangle's plane; edge vectors are linearly independent).

If for `var w = get_triangle_barycentric_coords(point, a, b, c)` the `w.x * a + w.y * b + w.z * c` is approx equal to `point` (approx because of the floats) then the returned `w` barycentric coords are correct.

Reply:
Yes, my mistake. I've updated it.

Reply:
Updated.